pkg/notes/document/document.go

```diff
@@ -230,8 +237,33 @@ func (d *Document) RenderMarkdownTemplate(bucket, fileDir, goTemplate string) (s
 	return s.String(), nil
 }
 
+// template returns either the default template or a template from file. The
+// `templateSpec` must be in the format of
+// `go-template:{default|path/to/template.txt`
+func (d *Document) template(templateSpec string) (string, error) {
+	if templateSpec == "go-template:default" {
+		return defaultReleaseNotesTemplate, nil
+	}
+
+	templatePath := strings.TrimPrefix(templateSpec, "go-template:")
+	if templatePath == templateSpec {
+		return "", errors.Errorf("bad template format: expected format %q, got %q", "go-template:path/to/file.txt", templateSpec)
+	}
+
+	b, err := ioutil.ReadFile(templatePath)
+	if err != nil {
+		return "", errors.Wrap(err, "reading template")
+	}
+	if len(b) == 0 {
+		return "", errors.Errorf("template %q must be non-empty", templatePath)
+	}
+
+	return string(b), nil
+}
+
 // RenderMarkdown accepts a Document and writes a version of that document to
 // supplied io.Writer in markdown format.
+// TODO: Deprecate uses of this method before adding deprecated warning.
```

Comment:
```suggestion
// Deprecated: prefer using the golang template instead of markdown
```

Reply:
I'll do this in a follow up under #1019. If I change it now then the test will fail.